### DISCUSSION TITLE
Welcome to Batch Antivirus Discussions!

### DISCUSSION THREAD

**Discussion by anic17** [anic17/Batch-Antivirus#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by NoEscapeEXE** [comment]
I was trying to make an antivirus out of batch before I saw this. And I was wondering.
How do you get the file hash of one file and compare it to another. This is all I need really.
I've been trying for so long looking on stackoverflow but I cant find one answer.
Please Help!!!

**Reply by Mora-Bytes** [reply]
for /f "tokens=2*" %%a in ('powershell Get-FileHash "%~1" -Algorithm sha256 ^| findstr /v /c:"Hash" /c:"----"') do (findstr /c:"%%a" "[DataBase]" && echo.'%~1' is malware || echo.'%~1' is not malware)

**Reply by Mora-Bytes** [reply]
yw
